dll/win32/browseui/internettoolbar.cpp

```diff
@@ -1716,6 +1716,7 @@ LRESULT CInternetToolbar::OnContextMenu(UINT uMsg, WPARAM wParam, LPARAM lParam,
 
     SHEnableMenuItem(contextMenu, IDM_TOOLBARS_LINKSBAR, FALSE);
 
+    pSettings->Load();
```

Comment:
I think the question of @JoachimHenze is like: "why putting this loading just in the middle of code that deals with setting up the menus, instead of before, or after?"
(and I would agree with this questioning)

Reply:
Moved.